Tests/MuxPlayerSwift/PlaylistURLLocalMapperTests.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  PlaylistLocalURLMapperTests.swift
+//
+//
+
+import Foundation
+import XCTest
+
+@testable import MuxPlayerSwift
+
+class PlaylistLocalURLMapperTests: XCTestCase {
+
+    func testMultivariantPlaylistLocalURLMapping() throws {
```

Comment:
nitpick(blocking): This is a media playlist test, not a multivariant playlist test.